lib/benchee/benchmark/scenario_context.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Benchee.Benchmark.ScenarioContext do
+  defstruct [
```

Comment:
I added the fast warning and run_time here pretty much just so I
had a consistent way of running warmups and real runs without having to have
additional information. It's essentially duplicating data from the config,
though. Is this a problem?